src/Microsoft.NetCore.Analyzers/Core/Security/DoNotDisableCertificateValidation.cs

```diff
@@ -0,0 +1,180 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using Analyzer.Utilities;
+using Analyzer.Utilities.Extensions;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Operations;
+
+namespace Microsoft.NetCore.Analyzers.Security
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
+    class DoNotDisableCertificateValidation : DiagnosticAnalyzer
+    {
+        internal const string DiagnosticId = "CA5359";
+        private static readonly LocalizableString Title = new LocalizableResourceString(
```

Comment:
Code style: static field names must prefixed with `s_`. So the field names should be `s_title`, `s_message`, etc.